common/test/acceptance/pages/studio/settings_advanced.py

```diff
@@ -262,7 +262,6 @@ def expected_settings_names(self):
             'teams_configuration',
             'social_sharing_url',
             'video_bumper',
-            'enable_proctored_exams',
```

Comment:
https://github.com/edx/edx-platform/commit/6f24c5a234a82d929304c3048cbf107dd36f897b seems a bad example, no matter here we remove or keep this expected field, all test cases pass anyway.